lib/matplotlib/_api/deprecation.py

```diff
@@ -29,7 +29,7 @@ def _generate_deprecation_warning(
             raise ValueError(
                 "A pending deprecation cannot have a scheduled removal")
     else:
-        removal = f"in {removal}" if removal else "two minor releases later"
+        removal = f"in {removal}" if removal else "two meso releases later"
```

Comment:
This is jargon; ideally we should parse `since` and compute the corresponding removal release accordingly (and add special-casing if we ever reach mpl4).
(FWIW the original code came during a major release transition and I was too lazy to add the special case for computing `removal` from `since`.